play-ahc-ws-standalone/src/main/resources/reference.conf

```diff
@@ -26,6 +26,8 @@ play {
       # Whether the raw URL should be used.
       disableUrlEncoding = false
 
+      # Whether to use LAX(no cookie name/value verification) or STRICT (verifies cookie name/value) cookie decoder
+      useLaxCookieEncoder = false
```

Comment:
We may want to be consistent with what we have on the server side:

https://github.com/playframework/playframework/blob/eb7483d73a5fa9a5ce51adb452663bf3a7999dbb/framework/src/play/src/main/resources/reference.conf#L121-L130

Reply:
done